issue_metrics.py

```diff
@@ -175,8 +175,12 @@ def get_per_issue_metrics(
                         issue_with_metrics.time_to_close = measure_time_to_close(
                             issue, None
                         )
+                if env_vars.hide_status is False:
```

Comment:
```suggestion
                if not env_vars.hide_status:
```

Reply:
Good catch! My new, new years resolution, "be more pythonic" 😆

Reply:
Good catch! My new years goal is now "be more pythonic" 😆